recipes-bsp/tools/setup-nv-boot-control/uefi_common.func.in

```diff
@@ -60,18 +60,15 @@ set_efi_var() {
 }
 
 gen_compat_2888() {
-    if [ "$fab" = "400" ]; then
-        if [ "$boardsku" = "0004" ]; then
-            boardrev=
+    if [[ "$boardsku" < "0004" ]]; then
```

Comment:
Looking at the `update_jetson_xavier_spec` function in the `nv-l4t-bootloader-config.sh` that comes in the latest L4T, what they have that we don't is:
```
    if [[ "${board_sku}" == "0004" ]]; then
        board_rev=""
        board_ver="400"
        return 0
    fi
```
so basically all boards with BOARDSKU of 0004 get treated as FAB 400.  Looks like this was added in V35.4.1.  Would that cover the boards you're seeing?  If so, then translating that into POSIX shell  and leaving the rest of this function as it was should align us with what's in L4T.

Reply:
If that's what NV is doing, we should too. Thanks for catching that, I'll align this.